jobs/grafana/templates/bin/prometheus-datasource

```diff
@@ -46,6 +46,9 @@ if [ -n "${DATASOURCE_ID}" ]; then
 else
   if curl -u ${GRAFANA_CREDENTIALS} -ksf -X POST ${GRAFANA_DATASOURCES_API} -H 'Content-Type: application/json' -d "${DATASOURCE_DATA}" ; then
     echo -e "\nCreated datasource ${DATASOURCE_NAME} at $(date)"
+  else
```

Comment:
It will only fail on create but not on updating an existing datasource. Can you also please this at [L43](https://github.com/cloudfoundry-community/prometheus-boshrelease/blob/6b8f327ef80067880e3c958aa9796ee0d627dd4f/jobs/grafana/templates/bin/prometheus-datasource#L43)?